.evergreen/download-crypt-shared-and-generate-sbom.sh

```diff
@@ -11,21 +11,10 @@ cat dist/.purls.txt
 
 set +x
 echo "${ARTIFACTORY_PASSWORD}" | docker login artifactory.corp.mongodb.com --username "${ARTIFACTORY_USERNAME}" --password-stdin
-cat << EOF > silkbomb.env
-SILK_CLIENT_ID=${SILK_CLIENT_ID}
-SILK_CLIENT_SECRET=${SILK_CLIENT_SECRET}
-EOF
 set -x
 
-trap_handler() {
-  rm -f silkbomb.env
-}
-trap trap_handler ERR EXIT
-
-docker pull artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:1.0
-docker run --rm -v ${PWD}:/pwd artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:1.0 update \
+docker pull artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:2.0
+docker run --rm -v ${PWD}:/pwd artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:2.0 update \
   --purls /pwd/dist/.purls.txt --sbom-out /pwd/dist/.sbom-lite.json
-docker run --env-file silkbomb.env --rm -v ${PWD}:/pwd artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:1.0 upload \
-  --silk-asset-group "${SILK_ASSET_GROUP}" --sbom-in /pwd/dist/.sbom-lite.json
-docker run --env-file silkbomb.env --rm -v ${PWD}:/pwd artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:1.0 download \
-  --silk-asset-group "${SILK_ASSET_GROUP}" --sbom-out /pwd/dist/.sbom.json
+docker run --env-file ${workdir}/kondukto_credentials.env --rm -v ${PWD}:/pwd artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:2.0 augment \
```

Comment:
```
[2025/02/19 23:51:24.089] + docker run --env-file /kondukto_credentials.env --rm -v /data/mci/4885e31066a9cc2d15804421224d7da2/src:/pwd artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:2.0 augment --repo mongodb-js/<REDACTED:notary_signing_key_name> --branch main_linux-x64 --sbom-in /pwd/dist/.sbom-lite.json -sbom-out /pwd/dist/.sbom.json
[2025/02/19 23:51:24.101] docker: open /kondukto_credentials.env: no such file or directory.
```

I'm guessing `${workdir}` is not defined here. Similar to the [comment for the Compass repo](https://github.com/mongodb-js/compass/pull/6734#discussion_r1963322606), I'd at least consider putting this into `/tmp` again.